spec/Dynamic-Validator.md

```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
 
 1. Pick **MAX_NUM_OF_VALIDATORS** candidates in order of having received many delegations.
 2. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
-3. Among the rest of them, drop the accounts having received less than MIN_CCS_RATE_TO_BE_VALIDATOR; the remains become validators.
+3. Among the rest of them, drop the accounts having received less than **DELEGATION_THRESHOLD**; the remains become validators.
+
+This process guarantes two things.
+* There are **MIN_NUM_OF_VALIDATORS** validators if the number of candidates are larger than **MIN_NUM_OF_VALIDATORS**.
+* The candidates that are not in **MIN_NUM_OF_VALIDATORS** and not receiving delegation more than **DELEGATION_THRESHOLD** will not be validators.
```

Comment:
* The candidates that are not in **MIN_NUM_OF_VALIDATORS** and not receiving delegation of more than **DELEGATION_THRESHOLD** will not be validators.